src/composables/useActions.ts

```diff
@@ -45,7 +45,21 @@ export function useActions() {
     execution: Transaction[]
   ) {
     if (!web3.value.account) return await forceLogin();
-    const pinned = await pin({ title, body, discussion, execution });
+
+    const executionData = execution.map(
+      (tx: Transaction, i: number): TransactionData => ({
+        ...tx,
```

Comment:
Do we want to remove `_type` and `form` from transaction in the stored file?

Reply:
No we should keep it to better display them after